src/model/digital_pass.py

```diff
@@ -66,6 +66,20 @@ def set_path(self, new_path: str):
     def voided(self):
         raise NotImplementedError()
 
+    @classmethod
+    def supported_mime_types(cls):
+        mime_types = []
+        for pass_type in cls.__subclasses__():
+            mime_types.append(pass_type.mime_type())
+        return mime_types
```

Comment:
```suggestion
        return [pass_type.mime_type() for pass_type in cls.__subclasses__()]
```
Same logic but using Python's list comprehensions.